### PR TITLE
Addition of public materials page/post

### PR DESCRIPTION
**Pull request by james-malone**
Adding new public materials page and blog post.

Staged @ http://104.197.97.73:4000

### CODE REVIEW

File: _pages/presentation_materials.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+We encourage you to use this content for public presentations, as the content has been contributed and is periodically reviewed by the Apache Beam community. When you use these slides, please remember to give credit to the authors of the presentations. Additionally, these presentations include speaker notes, you can view them by clicking **`View`** and selecting **`Show speaker notes`**.
+
+Some presentations are have `[Long]` or `[Short]` in their name. These tags are used when one topic, such as the Apache Beam Model, has both a long form and short form presentation.
```
Comment (takidau) [inline _pages/presentation_materials.md]: s/are have/have

**Comment by evilsoapbox** [comment]
Sorry for the commits - a few changes based on email feedback.

File: _posts/2016-04-03-presentation-materials.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+authors:
+  - fjp
+  - takidau
+---
+
+Are you interested in giving a presentation about Apache Beam? Perhaps you want to talk about Apache Beam at a local Meetup or a convention. Excellent!  The Apache Beam community is excited to expand and grow the community. To help kickstart this process, we are excited to announce an initial set of [Apache Beam presentation materials]({{ site.baseurl }}/presentation-materials/) which anyone can use to give a presentation about Apache Beam.
+
+<!--more-->
+
+This is the first of many steps the Apache Beam community will take to create high quality presentation materials. The Beam community has worked together to ensure these materials are accurate, up-to-date, and relevant. This initial release includes the following presentation materials:
```
Comment (francesperry) [inline _posts/2016-04-03-presentation-materials.md]: How about just: 

As a community, we want to build a shared collection of high quality presentation materials. This initial set includes the following slide decks:

File: _pages/presentation_materials.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Are you interested in learning more about Apache Beam or giving a talk about Apache Beam? Excellent! The Apache Beam community has created this collection of materials to help you get started. The community periodically reviews these materials to ensure they are accurate and up-to-date.
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+### Using these materials
+These materials are free to use! If you do use these materials, please remember to give credit to the authors of the original materials. These materials include speaker notes and you can view them by clicking **`View`** and selecting **`Show speaker notes`**. A set of presentations are have `[Long]` or `[Short]` in their name. Some topics, such as the *Apache Beam Model*, have two different presentations (a long and short version) depending on your needs and time constraints.
```
Comment (francesperry) [inline _pages/presentation_materials.md]: free to use -> can be used by anyone

Comment (davorbonaci) [inline _pages/presentation_materials.md]: I'd remove "of the original materials"; repetitive.

File: _pages/presentation_materials.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (davorbonaci) [inline _pages/presentation_materials.md]: The last two sentences are repetitive (A set of..., and Some topics, ...). I'd cut one (either), or merge them, or something along those lines.

File: _pages/presentation_materials.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+If you use these materials and questions arise:
+
+  * If anyone asks questions to which you are unsure of the answer, please do not hesitate to contact or direct the question to the [`user@beam.incubator.apache.org`](mailto:user@beam.incubator.apache.org) mailing list. The Apache Beam community is here to help.
```
Comment (davorbonaci) [inline _pages/presentation_materials.md]: The tone here might be improved. For example, you might say something like:

If you have any questions about the these materials, please ask on the user's mailing list. Any and all questions are welcome.

We welcome any and all feedback on these materials to improve them over time. If you deliver a presentation based on these, we'd love to hear from you on the user's mailing list. Please share with us your thoughts and, if available and appropriate, a link to the recording. In particular, we'd like to know questions you've been asked to incorporate them back into the materials.

File: _pages/presentation_materials.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  * The Beam community is also interested in knowing *common* questions which are asked. If you have time, please send some feedback to the [`user@beam.incubator.apache.org`](mailto:user@beam.incubator.apache.org) mailing list with questions which arose so we can help find ways to address these questions in the project documentation and these presentations.
+
+## Contributing new content
+The Apache Beam community is always looking for new core content and contributions! Specifically, the community is focused on developing new content in these areas:
```
Comment (davorbonaci) [inline _pages/presentation_materials.md]: Not sure about specifically listing additional content in the main page. This is fine for the blog, but we shouldn't track outstanding work on the site. (We have JIRA for that.)

So, if this needs tracking, we should have a JIRA, with appropriate labeling. Ok to link to the JIRA search, if needed. If it doesn't need tracking, I'd just be vague and say that we are always looking for new content. I'd probably do the latter.

File: _posts/2016-04-03-presentation-materials.md (hunk 2 above)
Comment (evilsoapbox) [inline _posts/2016-04-03-presentation-materials.md]: Done.

File: _pages/presentation_materials.md (hunk 3 above)
Comment (evilsoapbox) [inline _pages/presentation_materials.md]: Ack - what about "please remember to give credit to the authors cited in each presentation"

File: _pages/presentation_materials.md (hunk 4 above)
Comment (evilsoapbox) [inline _pages/presentation_materials.md]: Done - Updated and squashed the two together.

File: _pages/presentation_materials.md (hunk 5 above)
Comment (evilsoapbox) [inline _pages/presentation_materials.md]: Ack - change which incorporates this feedback coming in the next commit.

File: _posts/2016-04-03-presentation-materials.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+Are you interested in giving a presentation about Apache Beam? Perhaps you want to talk about Apache Beam at a local Meetup or a convention. Excellent!  The Apache Beam community is excited to expand and grow the community. To help kickstart this process, we are excited to announce an initial set of [Apache Beam presentation materials]({{ site.baseurl }}/presentation-materials/) which anyone can use to give a presentation about Apache Beam.
+
+<!--more-->
+
+As a community, we want to build a shared collection of high quality presentation materials. This initial set includes the following slide decks:
+
+* The Apache Beam model
+    * Long version
```
Comment (francesperry) [inline _posts/2016-04-03-presentation-materials.md]: Let's link these three directly to save folks a couple of clicks?

**Comment by francesperry** [comment]
LGTM from my side. Thanks, James!

File: _pages/presentation_materials.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+Are you interested in learning more about Apache Beam or giving a talk about Apache Beam? Excellent! The Apache Beam community has created this collection of materials to help you get started. The community periodically reviews these materials to ensure they are accurate and up-to-date.
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+## Using these materials
+These materials can be used by anyone. If you do use these materials, please remember to give credit to the authors cited in each presentation. These materials include speaker notes and you can view them by clicking **`View`** and selecting **`Show speaker notes`**. Some presentations are have `[Long]` or `[Short]` in their name, such as the *Apache Beam Model*, which means there are two different presentations (a long and short version) depending on your needs and time constraints.
```
Comment (takidau) [inline _pages/presentation_materials.md]: s/are have/have

**Comment by evilsoapbox** [comment]
All changes merged. Can someone please take a final look and merge? I still can't merge. :\

**Comment by davorbonaci** [comment]
Merged.

**Pull request closed without merging**